tests/routes/articles.spec.js

```diff
@@ -575,6 +577,26 @@ describe('ARTICLES TEST', () => {
       expect(res.body.totalRates).to.be.a('number');
       expect(res.body.rates).to.be.an('array');
     });
+
+    it('should get a specific article ratings ', async () => {
+      await db.Ratings.create({
```

Comment:
I think we should have a helper function for creating rating in the helper.js file in the test folder

Reply:
that makes sense so as to make it reusable. thank you